openoa/utils/plot.py

```diff
@@ -2118,7 +2118,7 @@ def plot_yaw_misalignment(
                 ],
                 color=curve_fit_color_code,
                 linestyle="--",
-                label=f"Max. Power Vane Angle = {round(curve_fit_params_ws[:,i,1].mean(),1)}$^\circ$",
+                label=f"Max. Power Vane Angle = {round(curve_fit_params_ws[:,i,1].mean(),1)}$^\circ$",  # noqa: W605
```

Comment:
nice that you found a fix for this! (assuming this resolves the error that gets thrown during the pre-commit workflow)

Reply:
That's exactly what this does!